### PR TITLE
add sentry_dsn and sign_up_url config options for oc-id

### PR DESCRIPTION
**Pull request by smith**

### CODE REVIEW

**Comment by mmzyk** [comment]
@smith And what happens when those values are empty? Currently they aren't set to anything in the attributes.

**Comment by smith** [comment]
These values get written out to config/settings/production.yml. If they are nil it falls back to the values in [config/settings.yml](https://github.com/chef/oc-id/blob/master/config/settings.yml). If those are nil or do not exist, the values are nil.

**Comment by mmzyk** [comment]
@smith So, I think that's great for someone who understand oc-id, but would be confusing for anyone else otherwise. I'd recommend putting default values in the attributes section, b/c as a Chef user that's where I'd expect to find them. Others could reasonably disagree I suppose.

File: files/private-chef-cookbooks/private-chef/recipes/oc_id.rb [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
-# Copyright:: Copyright (c) 2014 Chef, Inc.
+# Copyright:: Copyright (c) 2014-2015 Chef Software, Inc.
 #
 # All Rights Reserved
 
+# If no sign_up_url is defined, use the server URL.
+#
+# We don't have a clear way to detect whether Manage is installed or running or
+# whether it's running on an alternate host/port, short of slurping the
+# manage-running.json, so there's not an easy way to detect whay the *actual*
```
Comment (mmzyk) [inline files/private-chef-cookbooks/private-chef/recipes/oc_id.rb]: s/whay/what

**Comment by mmzyk** [comment]
Minor spelling error in a comment, otherwise :shipit:

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Pull request merged**